eng/tools/Directory.Build.props

```diff
@@ -1,3 +1,2 @@
 ﻿<Project>
```

Comment:
Can you add a comment that says "Intentionally empty to isolate tools projects from the result of the repo."